packages/test-commerce-sdk-react/app/pages/home.tsx

```diff
@@ -36,6 +37,13 @@ const Home = () => {
                     <Link to="/slas-helpers">useShopperLoginHelper</Link>
                 </li>
             </ul>
+
+            <h2>Miscellaneous</h2>
+            <ul>
+                <li>
+                    <Link to="/query-errors">Query errors</Link>
```

Comment:
Potentially a non-issue since it can be the expected behaviour:

I tested loading the page server-side only:
http://localhost:3000/products?__server_only

The terminal shows we render the page server-side twice. This is expected because when using the HoC `withReactQuery` we render the page two times using `react-ssr-prepass`.

**Question:** Is it expected after getting the error server-side to have the`isLoading` prop set to `true`?
Do we expect to render the error message server-side or do we still consider the request to be in a loading state despite getting an error server-side?
<img width="1792" alt="useProducts isLoading" src="https://user-images.githubusercontent.com/6024102/193700600-51c05a52-1797-414b-9296-b6518db9c056.png">

Reply:
Client-side I see the `console.log` displayed twice. Is that expected?

 <img width="1792" alt="Screen_Shot_2022-10-03_at_4_15_30_PM" src="https://user-images.githubusercontent.com/6024102/193702597-813f4230-50df-40c0-b31c-6e0ce7e16b22.png">

Reply:
From the 1st comment:

> Is it expected after getting the error server-side to have the `isLoading` prop set to true?

Hmm, good question. I don't know what the expected value is. On the client-side, it's set to false. So my guess is it has to be false on the server side too.

> Do we expect to render the error message server-side or do we still consider the request to be in a loading state despite getting an error server-side?

After the recent chats with Ben last time, what I do know is that where and how the error is rendered (either server or client side) is now in the hands of the developers. It's up to them... if they want to render the error:
- on server side with a generic error boundary: `useProducts({ids}, {useErrorBoundary: true}`
- on server side with custom error handling: call `useServerContext()` and/or `{error} = useProducts()`
- on client side: I guess that's possible too.. but why would they want to not render on _server_ side?

Reply:
From the 2nd comment above:

> Client-side I see the console.log displayed twice. Is that expected?

Yes, it's expected...  for now. On the client side, the same request is sent again. There's a [ticket](https://gus.lightning.force.com/lightning/r/ADM_Work__c/a07EE0000172lUtYAI/view) to make sure that there is no such duplicate request on the client side.